.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ on:
       - stable
 
 jobs:
-  test:
+  setup:
```

Comment:
This ends up being the name shown in the interface so `setup` is not that great. You can either rename this or add a `name` property I think